### DISCUSSION TITLE
Ring checkbox and radio button

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [tailwindlabs/tailwindcss#2818] [Help]
Hello, i'm trying some new features of tailwind 2.0. How can i remove totally the ring on checkbox and radio buttons?
I tried with .shadow-none.ring-0 and even with their focus variant, but if i focus the element there's still a small ring.
<img width="94" alt="Schermata 2020-11-19 alle 15 45 50" src="https://user-images.githubusercontent.com/63153191/99681818-e0c22d80-2a7e-11eb-82ab-b707ba386244.png">

**Comment by adamwathan** [comment]
`focus:ring-0` should do it — can you create a Tailwind Play that reproduces?

**Reply by federicocappellotto97** [reply]
Here you go: https://play.tailwindcss.com/0yVUAeBFOG
I'm using chrome on mac os big sur

**Comment by sergshumakov** [comment]
Try `focus:ring-0 focus:ring-offset-0`

**Comment by logmannn** [comment]
If you don't want any default rings then this is one way:
```
ringWidth: {
  'DEFAULT': '0px',
}
```
